buildroot/tests/run_tests

```diff
@@ -14,7 +14,11 @@ exec_test () {
     printf "\033[0;32mPassed\033[0m\n"
```

Comment:
GitHub won't let me comment on unmodified lines, so I'm actually referring to the line above this:
`if platformio run --project-dir $1 -e $2 --silent; then`

I think it would be useful to turn `--silent` into a parameter as well, like you did for `GIT_RESET_HARD`. It could be silent by default, but sometimes I want to see all the output.

Reply:
I'll add the option

Reply:
You can now set `VERBOSE_PLATFORMIO` to any value and it won't be `--silent`